src/components/ColumnSelector/ColumnSelector.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import { Meta, StoryObj } from "@storybook/react";
+
+import ColumnSelector from "./ColumnSelector";
+import MainTable from "components/MainTable";
+import {
+  getHeaders,
+  getRows,
+  removeHiddenColumns,
+  FILESYSTEM,
+  SNAPSHOTS,
+  USER_HIDEABLE_COLUMNS,
+  DESCRIPTION,
+  CLUSTER_MEMBER,
+} from "./ColumnSelectorStoriesHelper";
+
+const meta: Meta<typeof ColumnSelector> = {
+  component: ColumnSelector,
+  tags: ["autodocs"],
+};
+
+export default meta;
+
+type Story = StoryObj<typeof ColumnSelector>;
+
+export const Default: Story = {
+  render: () => {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const [userHidden, setUserHidden] = useState<string[]>([
+      FILESYSTEM,
+      SNAPSHOTS,
+      CLUSTER_MEMBER,
+    ]);
+
+    const sizeHidden = [DESCRIPTION];
+
+    const hiddenColumns = userHidden.concat(sizeHidden);
+    const ROWS = getRows();
+
+    const saveHidden = (columns: string[]) => {
+      localStorage.setItem("hiddenColumns", JSON.stringify(columns));
+    };
```

Comment:
I think we should remove the saving logic here. For pure display logic, this seems unnecessary.

We might move this into an exported function that can be used by apps and the story, but then need to make it customizable and tested. Not necessary from my side to have this in react-components. Wdyt?